--- a/mobile/android/geckoview/src/androidTest/java/org/mozilla/geckoview/test/util/RuntimeCreator.java
+++ b/mobile/android/geckoview/src/androidTest/java/org/mozilla/geckoview/test/util/RuntimeCreator.java
@@ -25,5 +25,5 @@ public class RuntimeCreator {
   public static final int TEST_SUPPORT_ERROR = 2;
   public static final String TEST_SUPPORT_EXTENSION_ID = "test-support@tests.mozilla.org";
-  private static final String LOGTAG = "RuntimeCreator";
+  private static final String LOGTAG = "RuntimeCreatorCRAMTEST";

   private static final Environment env = new Environment();
